### ISSUE TITLE
video fails to maintain aspect ratio on desktop monitors

### ISSUE THREAD

**Issue by warriordog** [ifd3f/ooo.eeeee.ooo#2]
When viewed on a desktop monitor (or any other device in landscape mode), the video fails to maintain aspect ratio. I suppose this could be intentional, but it results in a zoomed and pixelated video on wide monitors. See the below screenshots for an example:

![screenshot of the website in vertical form, showing the video in normal playback.](https://github.com/ifd3f/ooo.eeeee.ooo/assets/5315704/61c4970a-2c11-4dba-9b30-fe84db36a8f1)
![screenshot of the website in horizontal form, showing that the top / bottom of the image are cropped off.](https://github.com/ifd3f/ooo.eeeee.ooo/assets/5315704/7c7bf9c4-d289-45cb-98fc-db0ca6fca29f)

**Comment by ifd3f** [comment]
Yeah, I couldn't figure out a good solution for it that didn't also involve having white/black bars on the side. I guess I could do the thing most people do which is blur the really cropped video, and display the full video.
